django/core/models.py

```diff
@@ -1,8 +1,6 @@
 from uuid import uuid4
 
-from django.contrib.auth import get_user_model
 from django.db import models
-from django.utils import timezone
 
 
 class CollisionSafeFileField(models.FileField):
```

Comment:
Can rename this to `S3FileField` or `JoistFileField`.

Reply:
will do later. so far it was more about cleaning up things